packages/site/src/main.css

```diff
@@ -108,6 +108,6 @@ pre.root-pre {
   border-radius: var(--radius-base);
 }
 
-section[role="tabpanel"] > div > h1:first-of-type {
+section[role="tabpanel"]>div>h1:first-of-type {
```

Comment:
whitespace change from formatter - @scotttjob do we have an opinion on spaced vs no-spaced selector symbols?

Reply:
If asked, more white space is normally better. We can adjust the formatter to account for that.